src/lib/regions.ts

```diff
@@ -0,0 +1,49 @@
+import Client from '../client'
+
+interface RegionResponse {
+  name: string
+  code: string
+  latitude: number | null
+  longitude: number | null
```

Comment:
```suggestion
  latitude: number
  longitude: number
```

The corresponding go types don't seem to allow nulls https://github.com/superfly/flyctl/blob/master/api/types.go#L907

Is this intended?